### ISSUE TITLE
[Bug]: Bees pathfinding stuck at trying to fly through "Honey Glass (Player)"

### ISSUE THREAD

**Issue by TheFlyron** [Resourceful-Bees/ResourcefulBees#358]
### Bug Description

The Honey Glass (Player) correctly let's me pass through and blocks the bees. But the bees also get kind of stuck at the honey glass when trying to reach a flower outside of any building I create to keep them, observable in my own early-game constructions as in apiaries.

When I replace the Honey Glass (Player) by f.e. "Mob Filter (Player)" by Dark Utilities, I have functionally the same block and the bees don't try to fly through it all the time.

Perhaps for some reason the bees see the Honey Glass (Player) as passable block even if it is not? Is there a fix for that? Not every modpack with Resourceful Bees also has Dark Utilities. And I find the Honey Glass nicer to look at and work with.

### How to Reproduce?

1. Build an enclosed space with "Honey Glass (Player)" blocks as entrance.
2. Set free bees inside the space.
3. Place Bee Flowers (flowers/blocks) outside the enclosed space.
4. Bees constantly try to fly through the honey glass to no avail.

### Expected Behavior

Bees roam inside a bee building and don't crowd the honey glass entrance.

### Version

0.10.7

### Mod Loader Version

1.16.5 - 36.2.39

### Acknowledgement

- [X] I agree that I read the terms at the top of the page.

**Comment by TheFlyron** [comment]
Test setup:
![grafik](https://user-images.githubusercontent.com/13351617/197529079-b0072602-bd00-4f7f-9b62-dc5bb25e69a0.png)

1. An Iron Bee Nectarblock outside the enclosure.
2. A 1x2 door made out of Honey Glass (Player).
3. Any hive (or apiary)
4. Another Iron Bee Nectarblock inside the enclosure the bees only rarely find.
5. All the Iron Bees sticking to the Honey Glass to try reach the nectarblock outside.

(image taken in Sky Bees 2, current version: https://www.curseforge.com/minecraft/modpacks/sky-bees-2 )

For this case the solution is just "remove the outside nectarblock, duh". But what for the Cobblestone Bee? Not build anything out of cobblestone outside? Or any bee that flies to any #minecraft:flower.

**Comment by Epic428** [comment]
yeah so this is a known issue. for one the honey glass behaves _oddly_ with the bees and no matter how much debugging I've done I simply can't figure out how to actually correct it. what's more is I'm not entirely sure how it _isn't_ an issue with ethereal glass from glassentials since the code is practically the same. really the only difference is that we single out bees specifically rather than all mobs.

the other thing though is that bees should pathfind to the nearest "flower" however in some cases they may not and they may try to pathfind to a flower that's not accessible. the difference though is that with honey glass not involved the bee will eventually pick a new path. however bc honey glass **is** involved they detect that it doesn't have a collision box for the pathfinding and thus create a path through it, the collision box doesn't actually affect the bee until it collides with the block and prevents it from moving through. but yet I can't replicate the issue with ethereal glass and yet they have identical block properties.

It's particularly bizarre and if anyone has a solution to correcting it I would be fine with fixing it.
